index.php

```diff
@@ -1,3 +1,84 @@
 <?php
+  function parse_scripts($toJson = false) {
+    // scan directory for files matching pattern(s)
+    // store filename in object variable
+    $cli = [
+      'py' => 'python',
+      'js' => 'node',
+      'php' => 'php'
+    ];
+    
+    $regex = '/Hello World, this is ([\w\s]+) with HNGi7 ID HNG-(\d{5}) using (\w+) for stage 2 task/i';
```

Comment:
- [ ] digit scope: change `\d{5}` to `[0-9]{5}` since `\d` also includes underscore
 - [ ] capture group: expand `HNG-(\d{5})` to `(HNG-[0-9]{5})` to capture the entire `ID`
 - [ ] letter scope [optional]: change `\w` to `a-zA-Z` to restrict names to letters only

Reply:
@piouson \d doesn't is actually only digits `[0-9]`. I'll change `\w` you are right it includes underscores. I'll also change the ID capture group. Thanks. I didn't see this before my last push. Will fix them now